Ms/1095 history user not recorded in history objects (#1096)
acdh-oeaw/apis-core-rdf: docs/source/features.rst

for APIS. It is based on the `django-simple-history`_
package.

Track changes by users
^^^^^^^^^^^^^^^^^^^^^^

To record the user who made a change, the `django-simple-history` module 
used by APIS provides a middleware that sets the `history_user` attribute of 

> the request object to the current user. This attribute is then used by the 

Reviewer: ```suggestion
the request object to the current user. It then saves this information to the `history_user` attribute of the historical record. 
```

> `VersionMixin` to set the `history_user` attribute of the historical record. 

Reviewer: ```suggestion
```

To activate the middleware, add `simple_history.middleware.HistoryRequestMiddleware` 
to your `MIDDLEWARE` setting:

.. code-block:: python

    MIDDLEWARE = [
        ...

>         'apis_core.history.middleware.HistoryMiddleware',

Reviewer: ```suggestion
        'apis_core.history.middleware.HistoryRequestMiddleware',
```
